sentry-samples/sentry-samples-spring-boot/src/main/java/io/sentry/samples/spring/boot/CustomEventProcessor.java

```diff
@@ -0,0 +1,18 @@
+package io.sentry.samples.spring.boot;
+
+import io.sentry.core.EventProcessor;
+import io.sentry.core.SentryEvent;
+import org.jetbrains.annotations.Nullable;
+import org.springframework.stereotype.Component;
+
+/**
+ * Custom {@link EventProcessor} implementation lets modifying {@link SentryEvent}s before they are sent to Sentry.
+ */
+@Component
+public class CustomEventProcessor implements EventProcessor {
+  @Override
+  public SentryEvent process(SentryEvent event, @Nullable Object hint) {
+    event.setTag("Java Version", System.getProperty("java.version"));
```

Comment:
yep we could, but then i guess we would add it to sentry-core? I we go for it i need another idea for reasonable example of event processor ;-)

Reply:
not really, it should be a processor specifically for java backend/desktop, it should not be executed on Android at least, it won't return anything.
I mean it could live in sentry-core cus there's no compiling error, but I don't think it should be executed on runtime by default on all the platforms.

Reply:
Sentry does not recognize Java runtime. I keep it for now in a sample, we can consider moving it to `sentry-core` once it's agreed what runtime values are recognized.

Reply:
@maciejwalkowiak what do you mean by "Sentry does not recognize Java runtime"? if it's about the icon, this is just an UI limitation.
but agreed, we can make this in other PR, it doesn't make sense to block this one.

Reply:
Ah my bad. I thought unknown values are not passed but I just had NPE thrown in the processor. All good now :-) Anyway, I would prefer to close this one and move it to the next PR.